### ISSUE TITLE
No change

### ISSUE THREAD

**Issue by suhao16** [sunshangquan/logit-standardardization-KD#27]
I fixed the random seed first
Use commands
python tools/train.py --cfg configs/cifar100/kd/resnet32x4_resnet8x4.yaml --logit-stand --base-temp 2 --kd-weight 9

Modify the code in the file：  logit-standardization-KD\mdistiller\distillers\KD.py 
def normalize(logit):
    mean = logit.mean(dim=-1, keepdims=True)
    stdv = logit.std(dim=-1, keepdims=True)
    return (logit - mean) / (1e-7 + stdv)

def kd_loss(logits_student_in, logits_teacher_in, temperature, logit_stand):
    #logits_student = normalize(logits_student_in) #if logit_stand else logits_student_in
    #logits_teacher = normalize(logits_teacher_in) #if logit_stand else logits_teacher_in
    logits_student = logits_student_in
    logits_teacher = logits_teacher_in
    log_pred_student = F.log_softmax(logits_student / temperature, dim=1)
    pred_teacher = F.softmax(logits_teacher / temperature, dim=1)
    loss_kd = F.kl_div(log_pred_student, pred_teacher, reduction="none").sum(1).mean()
    loss_kd *= temperature**2
    return loss_kd
**Comment out normalize**
pip install -e .
python setup.py develop
Use same commands
python tools/train.py --cfg configs/cifar100/kd/resnet32x4_resnet8x4.yaml --logit-stand --base-temp 2 --kd-weight 9
There is no change in the accuracy of the first few epochs run.
Use commands
python tools/train.py --cfg configs/cifar100/kd/resnet32x4_resnet8x4.yaml  
There is change in the accuracy of the first few epochs run.

**Comment by suhao16** [comment]
--logit-stand does not work in 
def normalize（logit）：
mean = logit.mean（dim=-1， keepdims=True）
stdv = logit.std（dim=-1， keepdims=True）
return （logit - mean） / （1e-7 + stdv）

**Comment by suhao16** [comment]
> 我先修复了随机种子使用命令 python tools/train.py --cfg configs/cifar100/kd/resnet32x4_resnet8x4.yaml --logit-stand --base-temp 2 --kd-weight 9
> 
> 修改文件中的代码： logit-standardization-KD\mdistiller\distillers\KD.py def normalize（logit）： mean = logit.mean（dim=-1， keepdims=True） stdv = logit.std（dim=-1， keepdims=True） return （logit - mean） / （1e-7 + stdv）
> 
> def kd_loss（logits_student_in， logits_teacher_in， temperature， logit_stand）： #logits_student = normalize（logits_student_in） #if logit_stand else logits_student_in #logits_teacher = normalize（logits_teacher_in） #if logit_stand else logits_teacher_in logits_student = logits_student_in logits_teacher = logits_teacher_in log_pred_student = F.log_softmax（logits_student / 温度， dim=1） pred_teacher = F.softmax（logits_teacher / 温度， dim=1） loss_kd = F.kl_div（log_pred_student， pred_teacher， reduction=“none”）.sum（1）.mean（） loss_kd *= 温度**2 return loss_kd **注释掉 normalize** pip install -e . python setup.py develop 使用相同的命令 python tools/train.py --cfg configs/cifar100/kd/resnet32x4_resnet8x4.yaml --logit-stand --base-temp 2 --kd-weight 9 前几个 epoch 运行的准确性没有变化。使用命令 python tools/train.py --cfg configs/cifar100/kd/resnet32x4_resnet8x4.yaml 前几个 epoch 运行的准确性发生了变化。
